src/exampleTypes/ResultTable/index.js

```diff
@@ -63,7 +66,7 @@ let ResultTable = ({
     mutate,
     criteria,
   }
-  if (hasResults) {
+  if (!node.updating && hasResults) {
```

Comment:
Won't this mean that tables that are currently updating will just be blank?

Reply:
I think so. You should get the Loading indicator since the loaded wraps the component but the grid when loading would be displaying just the spinner. Do we want to have the results visible when the grid is updating? I can leave only the `hasResults` but it felt cleaner this way. I am fine either way.

Reply:
doesn't matter to me, as long as it was intentional